### ISSUE TITLE
Support for App Shortucts

### ISSUE THREAD

**Issue by pjakubczyk** [iBotPeaches/Apktool#1406]
### Information
1. **Apktool Version (`apktool -version`)** - v2.2.1
2. **Operating System (Mac, Linux, Windows)** - Mac
3. **APK From? (Playstore, ROM, Other)** - Github

### Stacktrace/Logcat

```
W: /Users/rudy/dev/projects/ShortcutDemp/app/build/outputs/apk/app-debug/res/xml-v25/shortcuts.xml:4: error: No resource identifier found for attribute 'shortcutId' in package 'android'
```

### Steps to Reproduce
1. `apktool b app-debug -o test.apk`

Latest apktool doesn't support Shortcuts API new xml tags specified in [documentation](https://developer.android.com/guide/topics/ui/shortcuts.html).

You can find sample project on my [github](https://github.com/pjakubczyk/shortcuts-demo) 

### APK
https://github.com/pjakubczyk/shortcuts-demo/blob/master/app-debug.apk

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? YES
2. If you are trying to install a modified apk, did you resign it? YES
3. Are you using the latest apktool version? YES

**Comment by xpirt** [comment]
[App Shortcuts](https://developer.android.com/guide/topics/ui/shortcuts.html) were added in Android 7.1 (API 25), therefore it needs 7.1 framework installed. 
Apktool's 2.2.1 internal framework is still 7.0 (API 24).

The current solutions are:
- Wait for Apktool 2.2.2, as this has already been fixed (7c6c7ab)
- Manually install a 7.1 framework with `apktool if` command (you can take it from any AOSP build)

**Comment by iBotPeaches** [comment]
As @xpirt said.
